### PR TITLE
adds superstimpaks, rework stimpaks, adds mutfruit, etc.

### PR DESCRIPTION
**Pull request by improvedname**
stimpak medipens now now include 15 units, they inject 5 units each time.
Stimpak healing has been buffed to cover all health. just a average quick heal.
superstimpaks are twice as effective.
fancy sprites to go with it 
https://i.imgur.com/FrtsSYH.gifv
https://i.imgur.com/QzsKVVd.gifv

Adds mutfruits. currently can be found in the megaseed vendor.
used to craft superstimpaks.

also kills off nuerotoxin/beepsky smash stun(again)

🆑 improvedname
add:mutfruit plants/seed to the ncr base and megaseed vendors
add:feracactus seeds to the legion base and megaseed vendors
add:some 308 clips to ncr starting crate
add:super stimpaks, and scribes start with them
tweak:changes throwing spears from normal size to huge, removing starting spears from legion loadout
refactor:some code
add:stimulant chem which removes stamina damage/stuns, but will apply stamina damage/oxyloss when overdosed
add:fancy sprites for the stimpak/superstimpak
del:removed stun effect from nuerotoxin/beepsky smash
del:removes service rifles from armoury(requested by ren)
del:removes r bolas from legion loadout(requested by ren)
tweak:tweaks stimpak metabolism rate(requested by ren)
tweak:buffs makeshift bolas breakout time(requested by ren)
add:heavy spear quiver
add:access locked NCR crates
del:removes NCR 5.57 magazines(requested by ren)
/🆑

### CODE REVIEW

**Comment by RenGusta** [comment]
Adjust to 5/5/4 for Stimpack.

10/10/8 for Super. If you're concerned about heal over time, you're welcome to discuss the metabolic rate.

File: tgstation.dme [hunk 1]
```diff
@@ -1647,6 +1647,7 @@
 #include "code\modules\hydroponics\grown\melon.dm"
 #include "code\modules\hydroponics\grown\misc.dm"
 #include "code\modules\hydroponics\grown\mushrooms.dm"
+#include "code\modules\hydroponics\grown\muttfruit.dm"
```
Comment (extramrdo) [inline tgstation.dme]: _mutt_fruit

**Comment by extramrdo** [comment]
Is it necessary to include the neurotoxin changes in a stimpack PR?

**Comment by JohnOxford** [comment]
sneak 100

File: tgstation.dme (hunk 1 above)
Comment (improvedname) [reply tgstation.dme]: Irrelevant

**Comment by improvedname** [comment]
@extramrdo Yes it is.
@JohnOxford what sneak 100? i litteraly said it in the description.

**Comment by improvedname** [comment]
Last pr i'm making.

**Comment by improvedname** [comment]
Oh yeah i forgot about the crate thing i made it objects and placed them into the map so its easier to change the loadout without messing with the maps

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -179,10 +179,38 @@
 	name = "stimpak"
 	desc = "A handheld delivery system for medicine, used to rapidly heal physical damage to the body."
 	icon_state = "stimpakpen"
-	volume = 10
-	amount_per_transfer_from_this = 10
-	list_reagents = list("stimpak" = 10)
-	
+	volume = 15
```
Comment (rhicora) [inline code/modules/reagents/reagent_containers/hypospray.dm]: This is overall a pretty severe buff for stimpaks, given that they not only heal more but each one comes with more fluid. Ren is requesting a nerf in the form of slower metabolization rate. Are we trying to make them better or worse?

File: code/game/objects/structures/crates_lockers/crates/f13crates.dm [hunk 3]
```diff
@@ -0,0 +1,85 @@
+/obj/structure/closet/crate/secure/weapon/ncr
+	name = "secure ncr crate"
+	desc = "Theres an NCR bear logo stamped on it"
+	req_access = list(ACCESS_SECURITY)
```
[the remark this answers is not in the document]
Comment (rhicora) [reply code/game/objects/structures/crates_lockers/crates/f13crates.dm]: Also, document this in your changelog.

**Comment by rhicora** [comment]
Were cotton and wheat seeds removed from Legion?

image

**Comment by rhicora** [comment]
Please document the other things you removed from the NCR armory.

image

File: code/game/objects/structures/crates_lockers/crates/f13crates.dm (hunk 3 above)
[the remark this answers is not in the document]
Comment (improvedname) [reply code/game/objects/structures/crates_lockers/crates/f13crates.dm]: yeah it works

File: code/modules/reagents/reagent_containers/hypospray.dm (hunk 2 above)
Comment (improvedname) [reply code/modules/reagents/reagent_containers/hypospray.dm]: 15 units, in injections of 5 ren is aware of this and gave me a metabolism number for this.

**Comment by improvedname** [comment]
@rhicora Legion seeds have an addition of feracactus, the rest is untouched
@rhicora NCR, basically has everything but lost their service rifles/service rifle magazines on ren's request

**Comment by UndeadNappist** [comment]
Please leave more reasoning when inquired than simply 'because someone said so.' That reason has been given in three PRs in the recent past, and they had to be either reverted or changed because they were erroneous or had different consequences than expected.

I will not be accepting that as adequate rationale in the future regardless of whom it comes from, including myself. If I attempt to to that, screenshot this and throw it in my face.

**Comment by improvedname** [comment]
I litteraly did what ren said, ask him about the changes.

**Pull request closed without merging**